docs/developers/CII-BestPractices-status.md

```diff
@@ -0,0 +1,742 @@
+</dd></dl></li>
+
+✅ **OCIO done**
+
+<li><a name="test_invocation"></a>A test suite SHOULD be invocable in a standard way for that language. <sup>[<a href="#test_invocation">test_invocation</a>]</sup><dl><dt><i>Details</i>:<dt> <dd>For example, "make check", "mvn test", or "rake test" (Ruby).</dd></dl></li>
+
+✅ **OCIO done**
+
+<li><a name="test_most"></a>It is SUGGESTED that the test suite cover most (or ideally all) the code branches, input fields, and functionality. <sup>[<a href="#test_most">test_most</a>]</sup></li>
+
+❓**OCIO needs somebody to assess**
+
+<li><a name="test_continuous_integration"></a>It is SUGGESTED that the project implement continuous integration (where new or changed code is frequently integrated into a central code repository and automated tests are run on the result). <sup>[<a href="#test_continuous_integration">test_continuous_integration</a>]</sup><dl><dt><i>Rationale</i>:<dt> <dd>See <a href="http://martinfowler.com/articles/continuousIntegration.html">Martin Fowler</a> There has been some shift in the meaning of the term continuous integration. Historically the term continuous integration focused on the first part - the frequent integration - and not on its testing.  However, over time the emphasis has shifted to include the notion of running automated tests as soon as the code is integrated.  We realize that this can be difficult for some projects to apply, which is why it is only SUGGESTED at the passing level.
+</dd></dl></li>
+
+❓**OCIO needs somebody to assess**
+
+</ul>
+
+<b><i>New functionality testing</i></b>
+
+<ul>
+
+<li><a name="test_policy"></a>The project MUST have a general policy (formal or not) that as major new functionality is added to the software produced by the project, tests of that functionality should be added to an automated test suite. <sup>[<a href="#test_policy">test_policy</a>]</sup><dl><dt><i>Details</i>:<dt> <dd>As long as a policy is in place, even by word of mouth, that says developers should add tests to the automated test suite for major new functionality, select "Met."</dd></dl></li>
+
+❓**OCIO needs somebody to assess**
+
+<li><a name="tests_are_added"></a>The project MUST have evidence that the <a href="#test_policy">test_policy</a> for adding tests has been adhered to in the most recent major changes to the software produced by the project. <sup>[<a href="#tests_are_added">tests_are_added</a>]</sup><dl><dt><i>Details</i>:<dt> <dd>Major functionality would typically be mentioned in the release notes. Perfection is not required, merely evidence that tests are typically being added in practice to the automated test suite when new major functionality is added to the software produced by the project.</dd></dl></li>
+
```

Comment:
I would say that there is an implicit rule (i.e. not documented) which implies to have unit tests for any new feature.